src/common/api.ts

```diff
@@ -49,6 +49,25 @@ class Fetcher {
         throw err;
       });
   }
+
+  public async put<T>(
+    url: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    data: { [key: string]: any },
+  ): Promise<ApiResponse<T>> {
+    return axios
```

Comment:
```suggestion
    return this.apiInstance
```

Il faut utiliser l'instance d'Axios qui est créée spécifiquement pour le client.